### PR TITLE
[test] Implement Cray linker workaround for CLE7.0.UP01.PS09

### PR DESCRIPTION
**Pull request by vkarak**
Fixes UES-829.

### CODE REVIEW

**Comment by vkarak** [comment]
@jenkins-cscs retry all

File: cscs-checks/compile/libsci_resolve.py [hunk 1]
```diff
@@ -76,6 +76,12 @@ def set_sanity(self):
                 sn.extractsingle(regex, self.stdout, 'version'), mod_name)
         ])
 
+    @rfm.run_before('compile')
+    def cray_linker_workaround(self):
```
Comment (ekouts) [inline cscs-checks/compile/libsci_resolve.py:80]: You haven't included the FIXME note in all the tests. Is this on purpose?

Comment (vkarak) [reply cscs-checks/compile/libsci_resolve.py:80]: Good point, I forgot it and I need to update the comment as well in the other fixme notes.

Comment (victorusu) [reply cscs-checks/compile/libsci_resolve.py:80]: @vkarak what do you think about adding this to XALT?

Comment (vkarak) [reply cscs-checks/compile/libsci_resolve.py:80]: @victorusu I don't understand this very well.

**Comment by vkarak** [comment]
@jenkins-cscs retry none

**Pull request merged**